uftrace.c

```diff
@@ -594,17 +594,14 @@ static int parse_option(struct opts *opts, int key, char *arg)
 
 	case 'A':
 		opts->args = opt_add_string(opts->args, arg);
-		opts->srcline = true;
 		break;
 
 	case 'R':
 		opts->retval = opt_add_string(opts->retval, arg);
-		opts->srcline = true;
 		break;
 
 	case 'a':
 		opts->auto_args = true;
-		opts->srcline = true;
 		break;
```

Comment:
could you let me know why above options enable srcline option?

Reply:
It was added for the record command to save debug info as much as possible.  But I think we can remove this part as you did, since the libmcount has a logic in `mcount_filter_init()` to control it properly.  Once it sets `needs_debug_info` it saves file and line number info.

Reply:
Thank you very much for the information.